src/mon/Elector.cc

```diff
@@ -1,4 +1,4 @@
-// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*- 
```

Comment:
Do we *have* to clean up whitespace in the whole file? I really hate messing up the git history that way...